tools/debugging/README.md

```diff
@@ -73,6 +73,17 @@ $ GST_DEBUG="GST_TRACER:7" GST_TRACERS="leaks(GstEvent,GstMessage)" \
 gst-launch-1.0 videotestsrc num-buffers=10 ! fakesink
 ```
 
+#### Case study 2: Tracing individual NNStreamer elements with GST_DEBUG and silent property
+
+Each NNStreamer elements has `silent` property. An NNStreamer element can be traced in verbose mode by setting `FALSE` to `silent` property.
+
+```bash
+$ GST_DEBUG="tensor_decoder:7" \
+gst-launch-1.0 fakesink ! tensor_decoder silent=FALSE ! fakesrc
```

Comment:
Is it a proper pipeline? sink->decoder->src huh?

Reply:
My bad. Though I wasn't meant to be a working example, the sink -> * -> src ordering doesn't make sense to me neigher 😅 .

I actually took the example from the comment here. I probably will make another PR to correct something similar to this in the source code.
https://github.com/nnstreamer/nnstreamer/blob/63d6fba3e441c7c514713c28f6309d8c013ccbee/gst/nnstreamer/tensor_decoder/tensordec.c#L38